catalog/confluence/catalog-confluence-source/pom.xml

```diff
@@ -125,6 +125,16 @@
             <artifactId>security-core-impl</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
❓ Should probably be adding config-api as well? I know it is pulled as a transitive dependency but for completeness sake.

Reply:
fixed